Activities/Java/UiPath.Java.Activities.ViewModels/UiPath.Java.Activities.ViewModels.csproj

```
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
+  <Import Project="..\Java.build.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net6.0-windows</TargetFrameworks>
+    <OutputPath>$(ProjectDir)..\..\Output\Activities\Java\</OutputPath>
+    <DefineConstants>VIEW_MODELS</DefineConstants>
+    <AssemblyName>UiPath.Java.Activities.ViewModels</AssemblyName>
+    <RootNamespace>UiPath.Activities.Java.ViewModels</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Resources\ActivitiesMetadataWindows.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\ActivitiesMetadataWindows.json" />
```

```suggestion
    <EmbeddedResource Include="Resources\ActivitiesMetadataWindows.json" LogicalName="$(RootNamespace).Resources.ActivitiesMetadata.json" />
```

Since we do not target net6-windows for activities, it is unclear to me as to how do we discern between Cross and Windows?

---

the change you pointed out is necessary
at the moment the activities are available on only on windows (even if java.activities is a cross-platform project and java in general is cross-platform)